Cookbook/Technical-Documents/How to use Slack?.md

```diff
@@ -0,0 +1,147 @@
+# How to use Slack?
+
+In Babylon we try to be as much remote friendly as we can.
+Slack is a tool we rely on a lot to help us communicate with everyone without having to be physically present.
+On the first day at Babylon, one of the first things you have automatically access is to your Slack account.
+Although this is a pretty straight forward instant messaging tool, there a few rules and tips everyone should be aware.
+
+## Which channels to join?
+
+### #ios
```

Comment:
Should we rename this channel to `#iOS-support` to match the ones used by other squads?